src/platform/arm-linux/iotjs_module_pwm-arm-linux.c

```diff
@@ -16,6 +16,10 @@
 
 #if defined(__linux__)
 
+#define _GNU_SOURCE // For asprintf
```

Comment:
There is no asprintf in this patch. Is it needed for other OSes ? If not, please remove ``#define`` clauses  or whole 3 lines.